src/tests/rnp_tests.cpp

```diff
@@ -41,7 +41,7 @@ rnp_tests::rnp_tests() : m_dir(make_temp_dir())
     /* We use LOGNAME in a few places within the tests
      * and it isn't always set in every environment.
      */
-    if (!getenv("LOGNAME")) {
+    if (!getenv_logname()) {
```

Comment:
This would slightly change previous behavior - since getenv_logname() may return non-NULL when `$USER` is set and $LOGNAME is unset. Most likely this will have no impact (and actually is on tests only) but it would be better to change it back.